tests/test_lazyflow/test_operators/test_opBaseFilter.py

```diff
@@ -0,0 +1,79 @@
+import numpy
+import pytest
+import vigra
+
+from lazyflow.operators.filterOperators import OpBaseFilter
+from lazyflow.roi import sliceToRoi
+from lazyflow.graph import InputSlot
+from lazyflow.rtype import SubRegion
+
+
+@pytest.fixture
+def dummy_filter_op(graph):
+    """OpBaseFilter that passes through input"""
+
+    def _filter_func(source, **filter_kwargs):
+        return source
+
+    class OpPassthroughFilter(OpBaseFilter):
+        _scale = InputSlot(value=0.5)
+        minimum_scale = 0.3
+
+        supports_window = True
+
+        name = "OpPassthroughFilter"
+        filter_fn = staticmethod(_filter_func)
+
+        def resultingChannels(self):
+            return 1
+
+    op = OpPassthroughFilter(graph=graph)
+    return op
+
+
+@pytest.mark.parametrize(
+    "slice1,computeIn2d",
+    [
+        (numpy.s_[0:1, 0:1, 0:1, 0:20, 0:30], True),
+        (numpy.s_[0:1, 0:1, 1:2, 0:20, 0:30], True),  # previous failure case
+        (numpy.s_[0:1, 0:1, 2:3, 0:20, 0:30], True),  # previous failure case
+        (numpy.s_[0:1, 0:1, 3:4, 0:20, 0:30], True),  # previous failure case
+        (numpy.s_[0:1, 0:1, 0:10, 0:1, 0:30], True),
+        (numpy.s_[0:1, 0:1, 0:10, 1:2, 0:30], True),
+        (numpy.s_[0:1, 0:1, 0:10, 2:3, 0:30], True),
+        (numpy.s_[0:1, 0:1, 0:10, 3:4, 0:30], True),
+        (numpy.s_[0:1, 0:1, 0:10, 0:20, 0:1], True),
+        (numpy.s_[0:1, 0:1, 0:10, 0:20, 1:2], True),
+        (numpy.s_[0:1, 0:1, 0:10, 0:20, 2:3], True),
+        (numpy.s_[0:1, 0:1, 0:10, 0:20, 3:4], True),
+        (numpy.s_[0:1, 0:1, 0:1, 0:20, 0:30], False),
+        (numpy.s_[0:1, 0:1, 1:2, 0:20, 0:30], False),
+        (numpy.s_[0:1, 0:1, 2:3, 0:20, 0:30], False),
+        (numpy.s_[0:1, 0:1, 0:10, 0:1, 0:30], False),
+        (numpy.s_[0:1, 0:1, 0:10, 1:2, 0:30], False),
+        (numpy.s_[0:1, 0:1, 0:10, 2:3, 0:30], False),
+        (numpy.s_[0:1, 0:1, 0:10, 3:4, 0:30], False),
+        (numpy.s_[0:1, 0:1, 0:10, 0:20, 0:1], False),
+        (numpy.s_[0:1, 0:1, 0:10, 0:20, 1:2], False),
+        (numpy.s_[0:1, 0:1, 0:10, 0:20, 2:3], False),
+        (numpy.s_[0:1, 0:1, 0:10, 0:20, 3:4], False),
+    ],
+)
+def test_single_slice_requests_first_slices(dummy_filter_op, slice1, computeIn2d):
```

Comment:
This also includes the case `(numpy.s_[0:1, 0:1, 3:4, 0:20, 0:30], False)`, which I assume should be covered too.

```suggestion
ref_slicing = numpy.s_[0:1, 0:1, 0:10, 0:20, 0:30]


@pytest.mark.parametrize(
    "slice1",
    [
        (*ref_slicing[:dim], slice(i, i + 1), *ref_slicing[dim + 1:])
        for dim in (2, 3, 4)
        for i in (0, 1, 2, 3)
    ]
)
@pytest.mark.parametrize("computeIn2d", [True, False])
def test_single_slice_requests_first_slices(dummy_filter_op, slice1, computeIn2d):
```

Reply:
yeah, a testament to how stuff is missed if not generated in code :)